lib/yocaml/index.mld

```diff
@@ -1,5 +1,7 @@
 {0 YOCaml}
 
+The full API can be {{:Yocaml/index.html} found here}
```

Comment:
Is it voluntary? I have nothing against it, it makes the link to the documentation easier to access. On the other hand, I'm thinking more and more of making a "tutorial" site because it's not easy to maintain the tutorial in the documentation (I think).

Reply:
Yes I always struggling to quickly find the full API. 
I agree with you, having a dedicated website could be better.

Reply:
Thanks.